src/Http/Client.php

```diff
@@ -549,3 +549,5 @@ protected function _createAuth($auth, $options)
         return new $class($this, $options);
     }
 }
+// @deprecated Backwards compatibility with earler 3.x versions.
+class_alias('Cake\Http\Client', 'Cake\Network\Http\Client');
```

Comment:
They shouldn't. The aliases just define additional classnames that resolve to the class on the left. It _could_ cause duplicate class name errors if userland code defined classes with the same names as those on the right of the `class_alias()` call.